barretenberg/cpp/src/barretenberg/stdlib/encryption/aes128/CMakeLists.txt

```diff
@@ -1 +1 @@
-barretenberg_module(stdlib_aes128 numeric stdlib_primitives)
\ No newline at end of file
+barretenberg_module(stdlib_aes128 numeric stdlib_primitives crypto_aes128)
```

Comment:
Needed to fuzz against a reference implementation and also for returning a value if all the inputs, key and iv are constant